Mr serverless ga (#17907)
cockroachdb/docs: src/current/cockroachcloud/plan-your-cluster-serverless.md

Storage for a multi-region cluster is billed at the same rate as a single-region cluster. However, by default data is replicated three times in the primary region and once in each additional region, and each replica in the additional regions will accrue more storage costs. For example, a three-region cluster with data replicated five times will use 5/3 times the storage space of a single-region cluster where data is replicated three times.

> Cross-region operations will use additional RUs because of the cost of cross-region networking. Cross-region networking costs depend on the source and destination regions. For read operations, the source region contains the replica and the destination region is the gateway region. For write operations, the source region is the gateway region and the destination region contains the replica. There is a network charge for each replica to which an operation writes. Refer to [Pricing](https://www.cockroachlabs.com/pricing) for a matrix of cross-region costs.

Reviewer: ```suggestion
Cross-region operations consume additional RUs for cross-region networking. Cross-region networking costs depend on the source and destination regions.

- For read operations, the source region contains the replica and the destination region is the gateway region.
- For write operations, the source region is the gateway region and the destination region contains the replica. There is a network charge for each replica to which an operation writes.

Refer to [Pricing](https://www.cockroachlabs.com/pricing) for a matrix of cross-region costs.
```

Keep in mind the following key points when planning your multi-region CockroachDB {{ site.data.products.serverless }} application's architecture:

- Write-heavy applications may experience a significant increase in RU consumption because replicating writes across all regions consumes more resources.
- Read-heavy applications may experience a smaller increase in RU consumption because the resources required to read from a single region of a multi-region cluster are comparable with a single-region cluster.

> - Cross-region reads are an anti-pattern and may significantly increase RU consumption. Features such as [global tables](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/global-tables.md), [regional by row tables](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/regional-tables.md), and [follower reads](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/follower-reads.md) make it possible to avoid most cross-region reads.

Reviewer: ```suggestion
- Cross-region reads are an anti-pattern and may significantly increase RU consumption. Features such as [global tables](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/global-tables.md), [regional by row tables](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/regional-tables.md), and [follower reads](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/follower-reads.md) help avoid most cross-region reads.
```

> - Cross-region writes will also consume additional RUs, but should not significantly increase consumption.

Reviewer: This feels like duplication from ~ line 65.
